python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -158,4 +163,44 @@ def configure_optimizers(self):
 )
 
 trainer.fit(model, cifar10_dm)
-trainer.test(model, datamodule=cifar10_dm)
+
+fp32_model = model
+# FP32 testing
+start = time()
+outputs = trainer.test(fp32_model, datamodule=cifar10_dm)
+time1 = time() - start
+log_dict = outputs[0]
+print("FP32 model size: %.3f MB" % fp32_model.model_size)
+print("Throughput: %d s" % time1)
+print("Accuracy: top1=%.2f%%" % (log_dict['test_acc'] * 100))
+
+# Save FP32 checkpoint
+trainer.save_checkpoint('ResNet18_on_CIFAR_FP32_CKPT.pth')
+
+# Run post-training quantization
+int8_model = trainer.quantize(fp32_model,
+                              calib_dataloader=cifar10_dm.train_dataloader(),
+                              val_dataloader=cifar10_dm.test_dataloader(),
+                              metric=Accuracy(num_classes=10),
+                              accuracy_criterion={'higher_is_better': True, 'relative': 0.1},
+                              raw_return=True)
+
+# compile quantized model as pytorch-lightning
+int8_model = trainer.compile(int8_model, loss=criterion, metrics=[accuracy])
+
+# Testing on quantized INT8 model
+start = time()
+outputs = trainer.test(int8_model, cifar10_dm)
+time2 = time() - start
+log_dict = outputs[0]
+print("INT8 model size: %.3f MB" % int8_model.model_size)
+print("Throughput: %d s" % time2)
+print("Accuracy: top1=%.2f%%" % (log_dict['test/Accuracy'] * 100))
+
+# Save INT8 checkpoint
+trainer.save_checkpoint('ResNet18_on_CIFAR_INT8_CKPT.pth')
+
+print("After quantization, the model size reduces %d%%, throughput improves %d%%" % (
+    (fp32_model.model_size - int8_model.model_size) / fp32_model.model_size * 100,
+    (time1 - time2) / time1 * 100
+))
```

Comment:
We may add an example for pytorch users as well? 
You may refer to https://github.com/TheaperDeng/git_learning/compare/mnist-quantization and it's an example copied from https://github.com/pytorch/examples/blob/master/mnist/main.py

Reply:
Sure. I can do that after this PR.

Reply:
Raised new issue here: https://github.com/intel-analytics/BigDL/issues/4031